signup.html

```diff
@@ -0,0 +1,116 @@
+    <title>Intuit account - Signup</title>
+    <link rel="stylesheet" href="./signup.css">
+    <script src="https://kit.fontawesome.com/e53eea7683.js" crossorigin="anonymous"></script>
+</head>
+<body>
+
+    <main>
+        <!-- Main contains the signin, the icons, and the form -->
+        <header>
+            <!-- This header contains the signin button -->
+            <div class="reset-account">
+                <!-- The reset account contains the reset password link-->
+                <p>Already have an intuit account?</p>
+                <a href="#">I forgot my User ID or Password</a>
+            </div>
+            <!-- The end of the reset account div-->
+            <button id="signin-button" type="submit">
+                <i class="fas fa-lock"></i> Sign In
+            </button>
+
+        </header>
+        <!-- The end of the header-->
+
+        <div class="Icons-form-c">
+            <!-- This contains the icons and the form -->
+            <div class="icons">
+                <!--This div will contain all 4 icons -->
+              <a href="https://accounts.intuit.com/signup.html" target="_blank" rel="noopener noreferrer">  <div id="intuit-icon">
+                    <!--This div will contain INTUIT icon -->
+                    <img src="./imgs/logo-intuit.png" alt="">
+                </div> </a>
+                <div class="customize-icons">
+                    <!-- This div will contain three circular logos -->
+                  <a href="https://mint.com" target="_blank" rel="noopener noreferrer">  <div class="company-logo">
+                        <img src="./imgs/mint.png" alt="">
+                        <p>Mint</p>
+                    </div> </a>
+                  <a href="http://quickbooks.intuit.com" target ="_blank" rel="noopener noreferrer">  <div class="company-logo">
+                        <img src="./imgs/logo-quickbooks.png" alt="">
+                        <p>quickbooks</p>
+                    </div></a>
+                    <a href="http://turbotax.intuit.com" target ="_blank" rel="noopener noreferrer"><div class="company-logo">
+                        <img src="./imgs/turbotax.png" alt="">
+                        <p>turbotax</p>
+                    </div></a>
+                </div>
+            </div>
+            <!-- The end of icons div -->
+        <div class="form-container">
+                <!--a huge container -> (h2,p)/form/ last(p)-->
+                <div class="center-content">
+                    <!-- div contains h2 and p tags-->
+                    <h2>Create an Intuit account</h2>
+                    <p>One account for everything Intuit, including Mint. <span><a href="#">learn more</a></span></p>
+                </div>
+                <!--end of (h2,p) div-->
+
+                <!--start of the form-->
+                <form>
+                    <label>Email address</label><br>
+                    <input type="text"><br><br>
```

Comment:
- [ ] **[MANDATORY]** Please use `type="email"` for this input to validate correctly
![image](https://user-images.githubusercontent.com/53256139/87631725-8e9e8380-c6fd-11ea-88cf-5f9d6dba6feb.png)